app/connectors/connectors/sql/src/main/resources/META-INF/syndesis/connector/sql.json

```diff
@@ -224,7 +224,7 @@
                 "secret": false,
                 "componentProperty": false,
                 "defaultValue": "",
-                "description": "StoredProcedure template to perform"
+                "description": "StoredProcedure template to perform."
```

Comment:
I don't know the context for this. I am wondering why it is "StoredProcedure" and not "Stored procedure" and then on line 284, I see "Stored Procedure template to perform." I expect they both need to be the same. But I don't know if the capital for procedure is needed because it is the name of the template.